glide.lock

```diff
@@ -271,6 +271,7 @@ imports:
   - kubernetes/typed/storage/v1/fake
   - kubernetes/typed/storage/v1beta1
   - kubernetes/typed/storage/v1beta1/fake
+  - pkg/api/v1
```

Comment:
No because pkg/volume/util depends on "k8s.io/kubernetes/pkg/api" specifically so we've no choice but to pick it up. Plus, pkg/api was kicked out of client-go and is supposed to live in k8s.io/api now.

Tangent:
The conundrum that is k8s + golang dependency management has been coming up a lot recently, it's probably time to document it properly (see https://github.com/kubernetes-incubator/external-storage/pull/237 and my halfhearted attempts at https://github.com/kubernetes-incubator/external-storage/tree/master/docs).

It used to be just client-go, then client-go + apimachinery, now there's a half dozen repos that must be in version or commit-hash sync. (btw, client-go has broken to some extent every quarter.) To be fair, glide and other tools should be smart enough to figure that dependency compatibility stuff out, and external-storage is a bit unique in that it has no choice but to chase after k8s master instead of using conveniently tagged versions like typical consumers.

I hope that doesn't sound like complaining, just want to write down openly somewhere my present thoughts, I'll work on documenting external-storage's vendoring "policies" next week. For the record, sig-api-machinery are the experts on all this stuff & from what I can tell are well aware of the unpleasantness that comes with writing an app for k8s. I haven't been following these developments closely enough, I guess they have to balance the needs of core k8s and k8s consumers.